### PR TITLE
Renamed UI `Vector1` to `Float` / Renamed enum `Float` precision to `Single`  (V2)

### PR DESCRIPTION
**Pull request by sebastienlagarde**
Replace PR: https://github.com/Unity-Technologies/Graphics/pull/1004 with a more up to date version

### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?

**This PR does mainly cosmetic changes for better undestanding for artists:**
* UI Only rename of `Vector 1` occurences to `Float`
* Renamed `Float` value of precision enums to `Single` (should do no halo as it's serialized as ints)

image

image

- Alias to our node searcher so Vector1 will continue to bring up the new float node. 
- Add a note about the change to our 20.2 upgrade guide

---
### Testing status

**Manual Tests**: What did you do?
- [ ] Opened test project + Run graphic tests locally
- [ ] Built a player
- [ ] Checked new UI names with UX convention
- [ ] Tested UI multi-edition + Undo/Redo + Prefab overrides + Alignment in Preset
- [ ] C# and shader warnings (supress shader cache to see them)
- [ ] Checked new resources path for the reloader (in developer mode, you have a button at end of resources that check the paths)
- Other: 

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/902-Graphics

Any test projects to go with this to help reviewers?

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

**Comment by sebastienlagarde** [comment]
So Yamato is green (all the failure are instability or already present on  master). The PR is good to go after @Unity-Technologies/gfx-qa-bellevue  and @sharlenet  review the doc.

@alindmanUnity have added the alias thing.

File: com.unity.render-pipelines.high-definition/Documentation~/Master-Node-Unlit.md [hunk 1]
```diff
@@ -27,10 +27,10 @@ The following table describes the input ports on an Unlit Master node, including
...
+| **Alpha**				                    | Float    | Fragment  | Defines the Material's alpha value. Used for transparency and/or alpha clip. Expected range 0 - 1.|
+| **AlphaClipThreshold**                    | Float    | Fragment  | Set the alpha value limit that HDRP uses to determine whether it should render each pixel of the Material. If the alpha value of the pixel is equal to or higher than this threshold then HDRP renders the pixel. If the value is lower than this threshold then HDRP does not render the pixel. This port only appears when you enable the **Alpha Clipping** setting.|
 | **Emission**                           	| Vector 3 | Fragment  | Defines the Material's emission color value. Expects positive values.|
-| **Shadow Tint**                              | Vector 4 | Fragment  | Defines the shadow color value with alpha value. Enable only if Shadow Matte enable cf. [Master node settings menu](#SettingsMenu)|
+| **Shadow Tint**                           | Vector 4 | Fragment  | Defines the shadow color value with alpha value. Enable only if Shadow Matte enable cf. [Master node settings menu](#SettingsMenu)|
```
Comment (sharlenet) [inline com.unity.render-pipelines.high-definition/Documentation~/Master-Node-Unlit.md:33]: I'm not quite sure what this means: "Enable only if Shadow Matte enable cf."

File: com.unity.shadergraph/Documentation~/Vector-1-Node.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
-# Vector 1 Node
```
Comment (sharlenet) [inline com.unity.shadergraph/Documentation~/Vector-1-Node.md:1]: I'd like to know whether we should rename this page to `Float.md` (as well as update all links to `Vector-1-Node.md`).

**Review by ghost — CHANGES_REQUESTED** [review]
When you create a float it says "Vector1" instead of float
image

**Review by ghost — APPROVED** [review]
accidental approval, please ignore

**Review by ghost — CHANGES_REQUESTED** [review]
image
image
"Single" under precision color mode should appear blue, but instead has no associated color

File: com.unity.shadergraph/Documentation~/Vector-1-Node.md (hunk 2 above)
Comment (sebastienlagarde) [reply com.unity.shadergraph/Documentation~/Vector-1-Node.md:1]: hmm... good point. We should rename it. Will you do it @sharlenet or you expect I do ?

File: com.unity.render-pipelines.high-definition/Documentation~/Master-Node-Unlit.md (hunk 1 above)
Comment (sebastienlagarde) [reply com.unity.render-pipelines.high-definition/Documentation~/Master-Node-Unlit.md:33]: this is not related to the change in this PR but it mean that Shadow tint property is only having an effect if shadow matte property is enabled.

**Comment by sebastienlagarde** [comment]
> When you create a float it says "Vector1" instead of float
> image

good catch, I fixed it. Not sure to understand what you mean by the "blue" for single. Was it blue  on master?

**Comment by sebastienlagarde** [comment]
> image
> image
> "Single" under precision color mode should appear blue, but instead has no associated color

@LandonTownsendUnity  so I checked on master and I have this:

image


I don't see the difference

File: com.unity.shadergraph/Documentation~/Vector-1-Node.md (hunk 2 above)
Comment (sharlenet) [reply com.unity.shadergraph/Documentation~/Vector-1-Node.md:1]: I've made the changes and will approve this PR.

File: com.unity.render-pipelines.high-definition/Documentation~/Master-Node-Unlit.md (hunk 1 above)
Comment (sharlenet) [reply com.unity.render-pipelines.high-definition/Documentation~/Master-Node-Unlit.md]: I rephrased it as follows:
> Defines the shadow color value with alpha value. The **Shadow Tint** property only has an effect if you enable the **Shadow Matte** property located in the [Master node settings menu](#SettingsMenu).

**Review by sharlenet — APPROVED** [review]

**Comment by sebastienlagarde** [comment]
> > When you create a float it says "Vector1" instead of float
> > image
> 
> good catch, I fixed it. Not sure to understand what you mean by the "blue" for single. Was it blue on master?

This is now fixed

**Review by marctem — APPROVED** [review]

**Review by cdxntchou — APPROVED** [review]
and search synonyms!

**Comment by sebastienlagarde** [comment]
Merged master with Antoine PR's change and fixed conflict. All good now. Yamato is running

**Comment by sebastienlagarde** [comment]
@LandonTownsendUnity ok, Antoine helped me to understand the issue with the blue line you where referening. All is fixed now. Reviewed, Yamatoed. So ready for your QA review. 
I have tested various cases. But I think we need to check migration with precision difference.

Thanks

image

**Review by ghost — APPROVED** [review]
Did some further testing, didn't find any other problems.

**Pull request merged**